package.json

```diff
@@ -3,27 +3,30 @@
   "description": "Front end application for the Digital Marketplace",
   "version": "0.0.1",
   "private": true,
-  "engine": "node >= 4.2",
+  "engine": "node == 6.10.3",
```

Comment:
Although this is what we use in production and it would be nice to lock this down, we need to think about how it might affect local development. If we lock this exact version, will it prevent using other versions - or just be silently ignored if that version isn't available?

It would be a good thing to get us all running on 6.10.3 but we need to be aware to send out a dev circular if things are suddenly going to stop working. Specifically thinking about Rob, for whom using 6.10.3 might not be as simple as it is for us on Macs. That's not to say we shouldn't do it of course, but let's be aware of the impact it'll have.

Reply:
Would something like ```^6``` be better?

We need to get better at updating our dependencies but you're right that we need to make sure we don't break anyone's local environment in the process.

Anyway I think we should be using at least v6.12.2 everywhere, so it would be better to decide now before I merge anything

https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V6.md

@risicle thoughts?

Reply:
It's not that it's "not simple" for me - it's that I practise proper package management rather than package soup. I have 6.11.5 available to me.

Reply:
So... if you look at the "security issues" addressed in the later v6 releases, they're actually security vulnerabilities in their *bundled copy of openssl*. Yes. nodejs *bundles their own copy of openssl* - what do they think they're playing at? Anyway, any sensible distribution patches that out and replaces it with the distribution's openssl. Thus nix's 6.11.5 is fully up to date with all security issues 6.12.whatever nodejs claims to be.

Reply:
I'll change it to ```"node >= 6.10.3"``` so it's less strict

Reply:
We should make sure it's a v6 though. Can you do something like `6.10.3 <= node < 7.0.0`?

Reply:
Good point @samuelhwilliams